TestON/tests/OnosCHO/OnosCHO.py

```diff
@@ -24,7 +24,10 @@ def CASE1( self, main ):
         onos-wait-for-start
         """
         import time
-
+        import imp
+        ThreadingOnos = imp.load_source('ThreadingOnos','/home/admin/ONLabTest/TestON/tests/OnosCHO/ThreadingOnos.py')
```

Comment:
I would suggest refactoring to main.Thread, and make sure I didn't break anything when I moved the class over. It should be as simple as doing a find & replace for ThreadingOnos.ThreadingOnos -> main.Thread